### ISSUE TITLE
misspelled unique field

### ISSUE THREAD

**Issue by kaizhu256** [strongloop/loopback-connector-mongodb#60]
https://github.com/strongloop/loopback-connector-mongodb/blob/master/lib/mongodb.js#L727

```
            options = {background: true};
            if(p.unique) {
-              options.unqiue = true;
+              options.unique = true;
            }
```

**Comment by gologo13** [comment]
this issue should be closed because #61 has already been merged.

**Comment by kaizhu256** [comment]
k
